spec

```diff
@@ -957,7 +957,6 @@ endseg
 beginseg
     name "ovl_Bg_Jya_Cobra"
     include "build/src/overlays/actors/ovl_Bg_Jya_Cobra/z_bg_jya_cobra.o"
-    include "build/data/overlays/actors/z_bg_jya_cobra.data.o"
     include "build/data/overlays/actors/z_bg_jya_cobra.reloc.o"
```

Comment:
Once everything in the file is at least non matching, auto generated reloc should be used when building with `NON_MATCHING`.
```suggestion
#ifdef NON_MATCHING
    include "build/src/overlays/actors/ovl_Bg_Jya_Cobra/ovl_Bg_Jya_Cobra_reloc.o"
#else
    include "build/data/overlays/actors/z_bg_jya_cobra.reloc.o"
#endif
```